src/reachy2_sdk/parts/arm.py

```diff
@@ -590,6 +625,9 @@ def _goto_single_joint(
             interpolation_mode=get_grpc_interpolation_mode(interpolation_mode),
         )
         response = self._goto_stub.GoToJoints(request)
+        if wait:
+            while not self._is_move_finished(response):
```

Comment:
maybe we can add a logger.info (.debug?) before and after the while since this can be quite long